test/Microsoft.ComponentDetection.Common.Tests/ComponentStreamEnumerableTests.cs

```diff
@@ -60,7 +64,12 @@ public void GetEnumerator_LogsAndBreaksEnumerationWhenFileIsMissing()
         var tempFileTwo = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
         var tempFileThree = Path.GetTempFileName();
         File.Delete(tempFileTwo);
-        this.loggerMock.Setup(x => x.LogWarning(Match.Create<string>(message => message.Contains("not exist"))));
+        this.loggerMock.Setup(x => x.Log(
+            LogLevel.Warning,
+            It.IsAny<EventId>(),
+            It.Is<It.IsAnyType>((v, t) => v.ToString().Contains(tempFileTwo)),
+            It.IsAny<IOException>(),
+            It.IsAny<Func<It.IsAnyType, Exception, string>>()));
```

Comment:
We briefly discussed creating an extension method, or other utility, to make this sort of setup and mocking easier. Not required for now, but nice-to-have.